include/wil/com.h

```diff
@@ -2777,9 +2776,9 @@ namespace wil
     @param source The string to write. Can be null if `writeLength` is zero
     @param writeLength The number of characters to write from source into `target`
     */
-    inline void stream_write_string(_In_ ISequentialStream* target, _In_reads_opt_(toWriteCch) const wchar_t*  source, _In_ size_t toWriteCch)
+    inline void stream_write_string(_In_ ISequentialStream* target, _In_reads_opt_(toWriteCch) const wchar_t*  source, _In_ size_t writeLength)

```

Comment:
Is this to fix a warning?  The `Cch` prefix has some value to denote that it is a character count and not a byte count.

Reply:
Different names between documentation and code, but of course I can revert this back

Reply:
I think it would be better to fix the documentation comment.